person.rb

```diff
@@ -3,14 +3,15 @@
 class Person
   attr_accessor :name, :age
 
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
   end
 
```

Comment:
A method for adding rentals should be present.

Link to an example [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)

```suggestion

def add_rental(person, date)
# Create new rental
end
```

Reply:
Ok.